install.py

```diff
@@ -28,6 +28,12 @@
 
 from func import *
 
+# Install needed library
+
+# django-axes
+os.system('pip install django-axes')
```

Comment:
I think this package should be installed by users. (Since it appears in our dependency list)